ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -22,7 +22,23 @@
               <Rose::Dropdown
                 @text={{t 'form.status.label'}}
                 @ignoreClickInside={{true}}
-              />
+                as |dropdown|
+              >
+                <Rose::Form::Checkbox::Group
+                  @items={{this.availableSessions}}
+                  @selectedItems={{this.selectedItems}}
+                  @onChange={{this.checkboxGroupChanged}}
```

Comment:
Is `checkboxGroupChanged` already defined? If this is a route action, we should call out that it's related to filters somehow. Thoughts on changing fn name to `onFilterSelection` or `filterUpdate`?

Reply:
this is a route action and can change to a better sounding name. We can go with `filterUpdate` to be in parity with other route actions